doc/userguide/rules/bypass-keyword.rst

```diff
@@ -1,19 +1,26 @@
 Bypass Keyword
 ==============
 
-Suricata has a ``bypass`` keyword that can be used in signatures to exclude traffic from further evaluation.
+.. role:: example-rule-action
+.. role:: example-rule-header
+.. role:: example-rule-options
+.. role:: example-rule-emphasis
```

Comment:
Do we need all these roles for this doc?

Reply:
We don't use them all now, I was thinking they would be imported for any future changes but can prune them in the next PR